mui-mappers/builder-mapper.js

```diff
@@ -35,6 +34,7 @@ const snapshotPropType = PropTypes.shape({ isDragging: PropTypes.bool }).isRequi
 const childrenPropType = PropTypes.oneOfType([PropTypes.node, PropTypes.arrayOf(PropTypes.node)]);
 
 const commonPropTypes = {
+  Component: PropTypes.func,
```

Comment:
This will fail if the component is class. We can use the `componentType` from the latest prop types for this type, but we have add the prop types to dependencies.

Reply:
It's already there. Fixed.